requirements/base.txt

```diff
@@ -1,4 +1,5 @@
 Jinja2
+jmespath
```

Comment:
`jmespath` is not a hard dependency for Salt, it's however in this case, a packaging hard dependency.

Move this to `requirements/static/pkg/{linux,windows,darwin,freebsd}.in`?
We should probably, eventually create a `requirements/static/pkg/common.in` ...